src/cli/generator/config.rs

```diff
@@ -28,14 +28,30 @@ pub struct Config<Status = UnResolved> {
 #[derive(Clone, Deserialize, Serialize, Debug, Default)]
 #[serde(rename_all = "camelCase")]
 pub struct PresetConfig {
-    merge_type: Option<f32>,
+    merge_type: Option<MergeType>,
     #[serde(rename = "consolidateURL")]
     consolidate_url: Option<f32>,
     use_better_names: Option<bool>,
     tree_shake: Option<bool>,
     unwrap_single_field_types: Option<bool>,
 }
 
+#[derive(Clone, Deserialize, Serialize, Debug, Default)]
+#[serde(rename_all = "camelCase")]
+pub struct MergeType {
+    threshold: f32,
+    merge_unknown_types: Option<bool>,
```

Comment:
This need not be a setting imo.

Reply:
actually this requirement is specified in issue.